metadata-ingestion/tests/integration/cassandra/test_cassandra.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+import pytest
+
+from datahub.ingestion.run.pipeline import Pipeline
+from tests.test_helpers import mce_helpers
+from tests.test_helpers.docker_helpers import wait_for_port
+
+logger = logging.getLogger(__name__)
+
+# TODO impl freezegun
+# FROZEN_TIME = "2024-01-01 10:00:00"
```

Comment:
When I set the `@freeze` header the requests to the cassandra docker instance timeout.
If I don't set it, everything works but the golden result is misaligned.
Was this issue encountered elsewhere? Any tips?